newIDE/app/src/Export/GenericExporters/OnlineWebExport/OnlineGamePropertiesDialog.js

```diff
@@ -0,0 +1,143 @@
+// @flow
+import { t, Trans } from '@lingui/macro';
+
+import * as React from 'react';
+import { type Game } from '../../../Utils/GDevelopServices/Game';
+import FlatButton from '../../../UI/FlatButton';
+import Dialog from '../../../UI/Dialog';
+import PublicGameProperties from '../../../GameDashboard/PublicGameProperties';
+import {
+  applyPublicPropertiesToProject,
+  type PartialGameChange,
+} from '../../../GameDashboard/PublicGamePropertiesDialog';
+import { getWebBuildThumbnailUrl } from '../../../Utils/GDevelopServices/Build';
+import RaisedButtonWithSplitMenu from '../../../UI/RaisedButtonWithSplitMenu';
+
+type Props = {|
+  project: gdProject,
+  saveProject: () => Promise<void>,
+  buildId: string,
+  game: Game,
+  onClose: () => void,
+  onApply: PartialGameChange => Promise<void>,
+  isLoading: boolean,
+|};
+
+export const OnlineGamePropertiesDialog = ({
+  project,
+  saveProject,
+  buildId,
+  game,
+  onClose,
+  onApply,
+  isLoading,
+}: Props) => {
+  const [name, setName] = React.useState<string>(project.getName());
+  const [categories, setCategories] = React.useState<string[]>(
+    project.getCategories().toJSArray()
+  );
+  const [description, setDescription] = React.useState<string>(
+    project.getDescription()
+  );
+  const [authorIds, setAuthorIds] = React.useState<string[]>(
+    project.getAuthorIds().toJSArray()
+  );
+  const [playWithKeyboard, setPlayableWithKeyboard] = React.useState<boolean>(
+    project.isPlayableWithKeyboard()
+  );
+  const [playWithGamepad, setPlayableWithGamepad] = React.useState<boolean>(
+    project.isPlayableWithGamepad()
+  );
+  const [playWithMobile, setPlayableWithMobile] = React.useState<boolean>(
+    project.isPlayableWithMobile()
+  );
+  const [orientation, setOrientation] = React.useState<string>(
+    project.getOrientation()
+  );
+  const [discoverable, setDiscoverable] = React.useState<boolean>(
+    !!game.discoverable
+  );
+  const thumbnailUrl = getWebBuildThumbnailUrl(project, buildId);
+
+  const saveProjectAndPublish = async () => {
+    await saveProject();
```

Comment:
Do you think you should try/catch around this line?

Reply:
the existing function already has a try catch / showErrorBox behavior built into it. So I think it's not necessary